### PR TITLE
Add stubs for winsound module

### PR DESCRIPTION
**Pull request by CraftSpider**

### CODE REVIEW

File: stdlib/2and3/winsound.pyi [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    SND_NOWAIT: int
+
+    MB_ICONASTERISK: int
+    MB_ICONEXCLAMATION: int
+    MB_ICONHAND: int
+    MB_ICONQUESTION: int
+    MB_OK: int
+
+    def Beep(frequency: int, duration: int) -> None: ...
+    def PlaySound(sound: Optional[Union[str, bytes, bytearray, memoryview]], flags: int) -> None: ...
```
Comment (reidswan) [inline stdlib/2and3/winsound.pyi]: PlaySound can accept a filename (I assume `str` is used for this :)) -- maybe add a `Path` to the `Union`?

Comment (srittau) [reply stdlib/2and3/winsound.pyi]: Actually `sound` must be a `str` or `None`:

```python
>>> winsound.PlaySound(b"", 0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'sound' must be str or None, not 'bytes'
```

(Tried with Python 3.8.0.)

Sidenote: `bytes` is equivalent to `Union[bytes, bytearray, memoryview]` in arguments.

Comment (reidswan) [reply stdlib/2and3/winsound.pyi]: My apologies, thank you for the correction @srittau

**Comment by CraftSpider** [comment]
`PlaySound` can accept a bytes-like object if passed the `SND_MEMORY` flag.

```py
>>> winsound.PlaySound(b"", winsound.SND_MEMORY)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: Failed to play sound
```
(I don't know off the top of my head a quick raw-bytes example in the format it wants, the main point is the lack of TypeError)

**Pull request merged**